src/Common/tests/Tests/System/BitOpsTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Test.Cryptography;
+
+using Xunit;
+
+namespace Internal.Runtime.CompilerServices
```

Comment:
`BitOps` imports this namespace (I am not sure why).
Is this workaround too hacky?